pkg/apis/hive/v1/validating-webhooks/clusterdeployment_validating_admission_hook.go

```diff
@@ -50,9 +50,13 @@ func NewClusterDeploymentValidatingAdmissionHook() *ClusterDeploymentValidatingA
 	if err != nil {
 		logger.WithError(err).Fatal("Unable to read managedDomains file")
 	}
-	logger.WithField("managedDomains", strings.Join(managedDomains, ",")).Info("Read managed domains")
+	domains := []string{}
+	for _, md := range managedDomains {
+		domains = append(domains, md.Domains...)
+	}
+	logger.WithField("managedDomains", strings.Join(domains, ",")).Info("Read managed domains")
```

Comment:
I don't think we need to do a `strings.Join` here. The formatting of the field will already display the domains.